packages/devtools/test/composite.test.ts

```diff
@@ -520,7 +531,39 @@ describe('composite', () => {
     expect(params).not.toEqual(third)
   })
 
-  test.todo('Composite.create()')
+  describe('Composite.create()', () => {
+    test('creates a new composite from valid schema', async () => {
+      const composite = await Composite.create({
+        ceramic: ceramic,
+        schema: compositeSchemaWithProfiles,
+        metadata: {
+          controller: ceramic.did.id,
+        },
+      })
+      expect(composite.hash).not.toBeFalsy()
+      const compositeParams = composite.toParams()
+      expect(Object.keys(compositeParams.commits).length).toEqual(3)
```

Comment:
Not an issue, no need to change anything, but here you could use `expect(...).toHaveLength(3)` instead